src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -461,6 +461,20 @@ public void timeEvent(final String eventName) {
     }
 
     /**
+     * Retrieves the time elapsed for the named event since timeEvent() was called.
+     *
+     * @param eventName the name of the event to be tracked that was previously called with timeEvent()
+     */
+    public double eventElapsedTime(final String eventName) {
+        final long currentTime = System.currentTimeMillis();
+        long startTime;
+        synchronized (mEventTimings) {
+            startTime = mEventTimings.get(eventName);
```

Comment:
We probably want to check if there is a timed event for the given name and return 0 in that case 👍